lib_xud/src/core/XUD_Main.xc

```diff
@@ -59,11 +59,13 @@ in port rx_rdy                 = PORT_USB_RX_READY;
 on USB_TILE: clock tx_usb_clk  = XS1_CLKBLK_4;
 on USB_TILE: clock rx_usb_clk  = XS1_CLKBLK_5;
 
-// We use a single array instrad of two here and append epAddr_setup on the end to save some instructions in the Setup 
+// We use a single array instrad of two here and append epAddr_Ready_setup on the end to save some instructions in the Setup 
 // token handling code. i.e. what we really want is the following, but's less efficient. 
-// unsigned epAddr[USB_MAN_NUM_EP]
-// unsigned epAddr[USB_MAX_NUM_EP_OUT]
-unsigned epAddr[USB_MAX_NUM_EP + USB_MAX_NUM_EP_OUT];
+// unsigned epAddr_Ready[USB_MAN_NUM_EP]
+// unsigned epAddr_Ready[USB_MAX_NUM_EP_OUT]
+unsigned epAddr[USB_MAX_NUM_EP];                        // Used to store the addr of each EP in ep_info array
+unsigned epAddr_Ready[USB_MAX_NUM_EP + USB_MAX_NUM_EP_OUT];    // Used by the EP to mark itself as ready, essentially same as epAddr_Ready with 0 entries.
```

Comment:
I don't understand this comment: this is `epAddr_Ready` and it says it's the "same as epAddr_Ready..."?

Reply:
Good spot, should just be `epAddr`